src/features/Post/postSlice.js

```diff
@@ -95,47 +111,59 @@ const postSlice = createSlice({
     },
     [getUserAllPosts.fulfilled]: (state, { payload }) => {
       state.allPosts = payload;
-      state.authStatus = "success";
+      state.postStatus = "success";
```

Comment:
suggestion: to use constants for values instead of hardcoing

Reply:
Sure will make the changes.